notebooks/experimental/ai_operators.ipynb

```diff
@@ -35,12 +35,12 @@
         "\n",
         "  <td>\n",
         "    <a href=\"https://colab.research.google.com/github/googleapis/python-bigquery-dataframes/blob/main/notebooks/experimental/ai_operators.ipynb\">\n",
-        "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Colab logo\"> Run in Colab\n",
+        "      <img src=\"../../third_party/logo/colab-logo.png\" alt=\"Colab logo\"> Run in Colab\n",
```

Comment:
Maybe you can upload the png files in another PR first, then use absolute paths instead?